components/index.d.ts

```diff
@@ -11,6 +11,7 @@ import { TypographyProps as TypographyProps$1 } from '@mui/material/Typography';
 import { ImageListItemProps } from '@mui/material/ImageListItem';
 import { ImageListProps } from '@mui/material/ImageList';
 import { AccordionProps } from '@mui/material/Accordion';
+import { ListProps as ListProps$1, ListItemProps as ListItemProps$1 } from '@mui/material';
```

Comment:
Well that's an interesting naming convention. I mean, MUI does do that underneath, but pretty funny to keep the convention on our side.